app/lib/applets/data_storage/definition.dart

```diff
@@ -11,12 +11,16 @@ final dataStorageDefinition = AppletDefinition(
   appletType: AppletType.navigation,
   icon: const Icon(Icons.folder_copy),
   selectedIcon: const Icon(Icons.folder_copy_outlined),
-  label: (context) => AppLocalizations.of(context)!.storage,
-  supportedAccountTypes: [AccountType.student, AccountType.teacher, AccountType.parent],
+  label: (context) => AppLocalizations.of(context)!.studyGroups,
```

Comment:
I think that should remain "storage"

Reply:
Will be fixed in next commit. Waiting for potential more fixes